src/cascadia/TerminalApp/Command.cpp

```diff
@@ -128,6 +130,10 @@ namespace winrt::TerminalApp::implementation
             {
                 result->_IterateOn = ExpandCommandType::Profiles;
             }
+            if (s == IterateOnSchemesValue)
+            {
+                result->_IterateOn = ExpandCommandType::ColorSchemes;
+            }
         }

```

Comment:
what you have here is a `JSON_ENUM_MAPPER(ExpandCommandType)`...!

Reply:
yeah! and it'll come with error checking for free

Reply:
line 126-137 become `JsonUtils::GetValueForKey(json, IterateOnKey, result->_IterateOn)` and it'll handle the reference/optional type and everything

Reply:
Okay good you are the real Dustin, had to leave that in to make sure you'd catch it 😜

Reply:
Ha!